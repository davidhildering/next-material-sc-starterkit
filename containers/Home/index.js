import React from "react";
import TopBanner from "../../components/TopBanner";
import MortgageCalculator from "../../components/MortgageCalculator";
import Contact from "../../components/Contact";
import { ProductExplanation } from "../../components/ProductExplanation";
import { FAQ } from "../../components/FAQ";
import { AboutUs } from "../../components/AboutUs";
import Benefits from "../../components/Benefits";
import NavBar from "../../components/NavBar";
import Footer from "../../components/Footer";
import styled from "styled-components";

export const CopyContainer = styled.div`
  padding-bottom: 24px;
  padding-top: 24px;
  min-height: 400px;
`;

class Home extends React.PureComponent {
  render() {
    return (
      <React.Fragment>
        <NavBar />
        <TopBanner />
        <Benefits />
        <ProductExplanation />
        <MortgageCalculator />
        <Contact />
        <FAQ />
        <AboutUs />
        <Footer />
      </React.Fragment>
    );
  }
}

export default Home;
