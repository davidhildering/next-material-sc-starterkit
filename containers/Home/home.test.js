import React from "react";
import { shallow } from "enzyme";
import Home from ".";

describe("Homepage", () => {
  it("should render correctly with no props", () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
