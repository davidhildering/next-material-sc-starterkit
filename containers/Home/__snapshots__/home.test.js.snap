// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should render correctly with no props 1`] = `
<Fragment>
  <NavBar />
  <WithStyles(TopBanner) />
  <Benefits />
  <ProductExplanation />
  <MortgageCalculator />
  <Contact />
  <FAQ />
  <AboutUs />
  <Footer />
</Fragment>
`;
